docs/src/main/tut/const.md

```diff
@@ -6,4 +6,278 @@ source: "https://github.com/non/cats/blob/master/data/src/main/scala/cats/data/C
 scaladoc: "#cats.data.Const"
 ---
 # Const
+At first glance `Const` seems like a strange data type - it has two type parameters, yet only
+stores a value of the first type. What possible use is it? As it turns out, it does
+have its uses, which serve as a nice example of the consistency and elegance of functional programming.
 
+## Thinking about `Const`
+The `Const` data type can be thought of similarly to the `const` function, but as a data type.
+
+```tut
+def const[A, B](a: A)(b: => B): A = a
+```
+
+The `const` function takes two arguments and simply returns the first argument, ignoring the second.
+
+```scala
+final case class Const[A, B](getConst: A)
+```
+
+The `Const` data type takes two type parameters, but only ever stores a value of the first type paramter.
+Because it never actually uses the second type parameter, it is referred to as a "phantom type."
```

Comment:
I've always thought so too -- it reminds me of the bad old days of `<b>broken <i>html</b> tags</i>`.